core/tee/tee_svc_storage.c

```diff
@@ -77,6 +77,8 @@ struct tee_svc_storage_head {
 	uint32_t head_size;
 	uint32_t meta_size;
 	uint32_t ds_size;
+	TEE_ObjectInfo info;
```

Comment:
This saves redundant or unneeded information in the persistent object. I would prefer to just add the needed fields from `TEE_ObjectInfo`.